lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -613,12 +613,12 @@ def test_single_problem(self, _get_current_task):
             metadata={'graded': True},
             display_name='Problem Vertical'
         )
-        self.define_option_problem(u'Pröblem1', parent=vertical)
```

Comment:
@cahrens I am going to take another look at this, probably on Tuesday or sooner depending on your comments.  It looks like this test set stores the same id for short and long and uses the problem url name in the id.  It is only failing on on these tests.  I am not sure why this set of tests was using the unicode character.  Please let me know what you think of these changes.